### PR TITLE
NAS-110575 / 12.0 / Stop TrueCommand connection on BACKUP node

### PR DESCRIPTION
**Pull request by themylogin**

### CODE REVIEW

**Comment by themylogin** [comment]
@william-gr sorry, we can't do it without you here :)

File: src/freenas/usr/local/lib/middlewared_truenas/plugins/failover_/event.py [hunk 1]
```diff
@@ -816,6 +816,7 @@ def carp_backup(self, fobj, ifname, vhid, event, user_override):
                     self.run_call('jail.stop_on_shutdown')
                     for vm in (self.run_call('vm.query', [['status.state', '=', 'RUNNING']]) or []):
                         self.run_call('vm.poweroff', vm['id'], True)
+                    await self.middleware.call('truecommand.stop_truecommand_service')
```
Comment (william-gr) [inline src/freenas/usr/local/lib/middlewared_truenas/plugins/failover_/event.py:819]: can we just use self.run_call here too, and do it before jail, please?

**Review by william-gr — APPROVED** [review]
Thanks!

**Comment by themylogin** [comment]
backport

**Pull request merged**